requirements.txt

```diff
@@ -1,17 +1,35 @@
+black==23.3.0
 certifi==2022.12.7
 charset-normalizer==3.1.0
+click==8.1.3
```

Comment:
you can also manage those in your pyproject.toml you'll add for packaging, example: https://github.com/PyBites-Open-Source/search/blob/main/pyproject.toml